app/views/reviews/_new.html.erb

```diff
@@ -0,0 +1,7 @@
+  
```

Comment:
can you put this header + form inside a div class=container to centre this on the view?

Reply:
Not in scope of current PR/